doc/overview.md

```diff
@@ -128,9 +128,11 @@ API token set up locally to be able to submit the results.
 
 ### Email Generator
 
-The Email Generator step listens for pub/sub events for when tests are complete
+The Email Generator step listens for pub/sub events when tests are complete
 to then generate a summary and send it via email.  The current implementation
-only prints the summary in the logs, and generates one for every test results
-received.  A production solution would send emails to various recipients
-depending on the tests and would wait for an event signalling that a group of
-tests has completed (or timed out).
+renders an email message based on the test report jinja template, opens a conenction
+to a STMP server and sends a report message to a user or a list of recipients once
+a change happened in the test results. At this moment emails are sent in case of a
+test fail or pass. A complete production solution would send emails to various
+recipients depending on the tests and would wait for an event signalling that
+a group of tests has completed (or timed out).
```

Comment:
```suggestion
a group of tests has been completed (or timed out).
```

Reply:
I am not sure about this as well, but I'd like to mean that the test was completed or even time out, and to use been I will need to update to something like: "has been completed or has been timed out". 

But I don't know, I am not a native speaker of English, maybe @gctucker can give us a hand.